src/Taweret/tests/test_trees.py

```diff
@@ -81,6 +80,9 @@ def test_mixing():
         adaptevery=500,
         minnumbot=4)
 
+    # Add to satisfy .flake8
+    fit.keys()
```

Comment:
Did flake8 complain because the `fit` variable was set but not used?  If so, is it OK that the test doesn't actually test the result directly?  Rather than add in a do nothing line, you should be able to call `mix.train` without saving its return variable to a local variable in the test (i.e., remove `fit = `).